lib/src/dbs/capabilities.rs

```diff
@@ -177,7 +177,8 @@ impl<T: Target + Hash + Eq + PartialEq + std::fmt::Display> std::fmt::Display fo
 /// # Examples
 ///
 /// Create a new instance, and allow all capabilities
-/// ```no_run
+#[cfg_attr(feature = "kv-rocksdb", doc = "```no_run")]
+#[cfg_attr(not(feature = "kv-rocksdb"), doc = "```ignore")]
```

Comment:
These doc tests fail if `kv-rocksdb` isn't enabled. These attributes make the doc-test not run if the feature is not enabled.

Reply:
I think this might be a good change, but lets do it in a different PR.